conf/nginx/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nginx
+
+RUN rm /etc/nginx/conf.d/default.conf
+ADD recommendation.conf /etc/nginx/conf.d/
```

Comment:
Same disclaimers as above:
> **DL3006** Always tag the version of an image explicitely.
> **DL4000** Specify a maintainer of the Dockerfile
> **DL3020** Use COPY instead of ADD for files and folders